insolar/record.go

```diff
@@ -76,11 +76,8 @@ func (id *ID) Hash() []byte {
 }
 
 // Equal checks if reference points to the same record.
-func (id *ID) Equal(other *ID) bool {
```

Comment:
Why? It breaks business logic.

Reply:
It's something that is demanded by `gogo` marshaller. It passes ID as object, not pointer.
I've found every place in code and changed it appropriately.